specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineImage.json

```diff
@@ -665,6 +665,59 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/edgeZones/{edgeZone}/vmimages": {
+      "get": {
+        "tags": [
+          "VirtualMachineImages"
+        ],
+        "operationId": "VirtualMachineImagesEdgeZone_ListVMImagesByEdgeZone",
```

Comment:
"VirtualMachineImages_ListByEdgeZone

Reply:
What should I change tags or operationId?

Reply:
operation id pls